transit/rest_api/superset/superset_auth.py

```diff
@@ -0,0 +1,96 @@
+import json
+import os
+import requests
+
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.http import require_POST, require_GET
+
+from rest_framework import (
+    permissions,
+    status
+)
+from rest_framework.decorators import (
+    api_view,
+    authentication_classes,
+    permission_classes
+)
+from rest_framework.response import Response
+from rest_framework_simplejwt import authentication
+
+
+SUPERSET_HOST = os.getenv('SUPERSET_HOST', 'http://localhost:8088/')
```

Comment:
I think we should move this env variables to settings.py

Reply:
ok, Good point. I will move those env into setting.py.

Reply:
fixed